<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Senritsu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1e1e1e;
}
QWidget {
    font-family: &quot;Roboto&quot;, sans-serif;
    color: #ffffff;
}
QPushButton {
    color: #ffffff;
    background-color: #333333;
    border: 1px solid #555555;
    padding: 8px;
    border-radius: 5px;
    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
    transition: transform 0.1s ease-in-out;
}
QPushButton:hover {
    background-color: #444444;
    transform: scale(1.05);
}
QPushButton:pressed {
    background-color: #007ACC;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_File_Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #E1B7C4;
color: black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Audio file Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_File_Name">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C04563;
color: black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>781</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>12</number>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>       Volume:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SongListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>781</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg.ico);
background-repeat: no-repeat;
background-position: center;
background-color:   #F5F5F5;
border-radius: 10px;
color: black;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:   #F5F5F5; border: 1px solid #B3E5FC; border-radius: 5px;
color :black;</string>
    </property>
    <property name="placeholderText">
     <string>Search....</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="openVideoAction"/>
    <addaction name="stopVideoAction"/>
    <addaction name="addBiblioSongAction"/>
    <addaction name="updateSongAction"/>
    <addaction name="exitAppAction"/>
   </widget>
   <widget class="QMenu" name="playlistMenu">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="addPlaylistAction"/>
    <addaction name="savePlaylistAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAppAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="playlistMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="exitAppAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="addPlaylistAction">
   <property name="text">
    <string>New Playlist</string>
   </property>
  </action>
  <action name="aboutAppAction">
   <property name="text">
    <string>Abouts us</string>
   </property>
  </action>
  <action name="savePlaylistAction">
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
  <action name="updateSongAction">
   <property name="text">
    <string>Update Biblio</string>
   </property>
  </action>
  <action name="addBiblioSongAction">
   <property name="text">
    <string>Add Biblio</string>
   </property>
  </action>
  <action name="openVideoAction">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="stopVideoAction">
   <property name="text">
    <string>Stop Video</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
